src/osm_configurator/view/toplevelframes/main_menu_frame.py

```diff
@@ -35,35 +38,71 @@ def __init__(self, state_manager: StateManager, project_controller: IProjectCont
+                         height=frame_constants_i.FrameConstants.HEAD_FRAME_HEIGHT.value,
+                         corner_radius=frame_constants_i.FrameConstants.FRAME_CORNER_RADIUS.value,
+                         fg_color=frame_constants_i.FrameConstants.HEAD_FRAME_FG_COLOR.value)
 
         self._project_controller = project_controller
         self._state_manager = state_manager
-
         self._passive_projects = []
 
-        # Implementing the buttons
-
-        customtkinter.CTkButton(self, text="New Project", command=self.__create_project) \
-            .grid(row=0, line=0, padx=40, pady=40)
+        # Configuring the grid
+        self.grid_columnconfigure(0, weight=1)
+        self.grid_columnconfigure(1, weight=1)
+        self.grid_columnconfigure(2, weight=1)
+        self.grid_columnconfigure(3, weight=1)
+        self.grid_columnconfigure(4, weight=1)
+        self.grid_rowconfigure(0, weight=1)
+        self.grid_rowconfigure(1, weight=2)
+        self.grid_rowconfigure(2, weight=2)
+        self.grid_rowconfigure(3, weight=1)
 
-        customtkinter.CTkButton(self, text="Load external Project") \
-            .grid(row=1, line=0, padx=40, pady=40)
-
-        customtkinter.CTkButton(self, text="Load selected Project") \
-            .grid(row=2, line=0, padx=40, pady=40)
+        # Implementing the buttons
 
-        customtkinter.CTkButton(self, text="Settings", command=self.__call_settings()) \
-            .grid(row=3, line=0, padx=40, pady=40)
+        self.main_buttons_left = [
+            customtkinter.CTkButton(master=self,
+                                    text="New Project",
+                                    command=self.__create_project),
+            customtkinter.CTkButton(master=self,
+                                    text="Load external Project",
+                                    command=self.__load_external_project),
+            customtkinter.CTkButton(master=self,
+                                    text="Settings",
+                                    command=self.__call_settings)
+        ]
+
+        # Align the buttons and give them the standard style-attributes
+        for i, button in enumerate(self.main_buttons_left):
+            button.configure(
+                border_width=button_constants_i.ButtonConstants.BUTTON_BORDER_WIDTH.value,
+                fg_color=button_constants_i.ButtonConstants.BUTTON_FG_COLOR_ACTIVE.value,
+                hover_color=button_constants_i.ButtonConstants.BUTTON_HOVER_COLOR.value,
+                border_color=button_constants_i.ButtonConstants.BUTTON_BORDER_COLOR.value,
+                text_color=button_constants_i.ButtonConstants.BUTTON_TEXT_COLOR.value
+            )
+
+            button.grid(row=i, column=0, rowspan=1, columnspan=1, padx=10, pady=10)
 
         # showing all entries in custom boxes
-        for passive_project in self._passive_projects:
+        for i, passive_project in enumerate(self._passive_projects):
```

Comment:
this loop doesn't look correct, can you do (i, something) in a loop?

Reply:
Yes